source/features/edit-comments-faster.tsx

```diff
@@ -12,14 +12,19 @@ function init(): void {
 		add(comment) {
 			comment.classList.add('rgh-edit-comment');
 
+			let discussionsClassname = '';
+			if (pageDetect.isDiscussion()) {
+				discussionsClassname = 'js-discussions-comment-edit-button';
+			}
+
 			comment
 				.closest('.js-comment')!
-				.querySelector('.timeline-comment-actions > details:last-child')! // The dropdown
+				.querySelector('.timeline-comment-actions > details:last-child, .timeline-comment-actions details:last-child')! // The dropdown
 				.before(
 					<button
 						type="button"
 						role="menuitem"
-						className="timeline-comment-action btn-link js-comment-edit-button rgh-edit-comments-faster-button"
+						className={`timeline-comment-action btn-link js-comment-edit-button ${discussionsClassname} rgh-edit-comments-faster-button`}
```

Comment:
Does `js-comment-edit-button` exist on disscussions?
```suggestion
						className={`timeline-comment-action btn-link ${pageDetect.isDiscussion() ? 'js-discussions-comment-edit-button' : 'js-comment-edit-button'} rgh-edit-comments-faster-button`}
```

Reply:
Yes:
![2021-05-09_170639](https://user-images.githubusercontent.com/723651/117575080-f2f55e00-b0e8-11eb-9507-95064bd0ad77.jpg)

Reply:
I added your suggestion in my latest commit, thank you.